server/api/crud/model_monitoring/deployment.py

```diff
@@ -1110,20 +1121,18 @@ def set_credentials(
                 or _default_secrets_v3io
             )
         if endpoint_store_connection:
-            if (
-                endpoint_store_connection != "v3io"
-                and not endpoint_store_connection.startswith("mysql")
-                and not endpoint_store_connection.startswith("sqlite")
+            if not endpoint_store_connection.startswith(
+                mlrun.common.schemas.model_monitoring.ModelEndpointTargetSchemas.list()
```

Comment:
`startswith` only works with str or tuple, not list
```suggestion
                tuple(mlrun.common.schemas.model_monitoring.ModelEndpointTargetSchemas.list())
```